Test/TestsDatabase/OrderDatabaseTests.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Linq;
+using Anlab.Core.Domain;
+using AnlabMvc.Data;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Internal;
+using Shouldly;
+using Test.Helpers;
+using Xunit;
+
+namespace Test.TestsDatabase
+{
+    [Trait("Category", "DatabaseTests")]
+    public class OrderDatabaseTests
+    {
+        [Fact]
+        public void test1()
+        {
+            var connection = new SqliteConnection("DataSource=:memory:");
+            connection.Open();
+
+            try
+            {
+                var options = new DbContextOptionsBuilder<ApplicationDbContext>()
+                    .UseSqlite(connection)
+                    .Options;
+
+                using (var context = new ApplicationDbContext(options))
+                {
+                    context.Database.EnsureCreated();
+                }

```

Comment:
@srkirkland It was in the test example:
https://docs.microsoft.com/en-us/ef/core/miscellaneous/testing/sqlite